armonik/admin-0-8-gui.tf

```
@@ -1,29 +1,29 @@
 # Control plane deployment
-resource "kubernetes_deployment" "admin_old_gui" {
-  count = var.admin_old_gui != null ? 1 : 0
+resource "kubernetes_deployment" "admin_0_8_gui" {
+  count = var.admin_0_8_gui != null ? 1 : 0
   metadata {
-    name      = "admin-old-gui"
+    name      = "admin-0_8-gui"
```

nitpick: I would prefer dash:
```suggestion
    name      = "admin-0-8-gui"
```

---

Should I change it in every name where a dash was already there?